### DISCUSSION TITLE
[QUESTION] Longhorn 1.4.x and Kubernetes 1.26

### DISCUSSION THREAD

**Discussion by aureq** [longhorn/longhorn#5596] [General]
## Question

The latest release page states that k8s v1.21 should be the minimum supported version. However, it is unclear what is the maximum k8s version supported. Is it v1.25 or v1.26? And as v1.27 isn't far away, it would be great to have a tracking issue for this too.

Could it be possible for someone to update the docs ([1](https://longhorn.io/docs/1.4.1/deploy/upgrade/), [2](https://longhorn.io/docs/1.4.1/deploy/install/)) on the maximum k8s supported version.

Thank you 💜

## Environment

 - Longhorn version: 1.4.0
 - Kubernetes version: 1.25.6+k3s1
 - Node config
   - OS type and version; debian
   - CPU per node: 4/8
   - Memory per node: 32G
   - Disk type: nvme
   - Network bandwidth and latency between the nodes: 1gbps
 - Underlying Infrastructure: Baremetal

**Comment by derekbit** [comment]
cc @weizhe0422 @PhanLe1010 @innobead

**Comment by innobead** [comment]
From https://kubernetes.io/docs/reference/using-api/deprecation-guide/, there are no incompatible risks for 1.4 until 1.29 which is the latest info on the page. However, we will review this again while doing the K8s version bump for 1.5.0. 

Please follow up https://github.com/longhorn/longhorn/issues/5595. 

cc @PhanLe1010
